mobile-app/app/components/SummaryTitleV2.tsx

```diff
@@ -0,0 +1,121 @@
+import BigNumber from 'bignumber.js'
+import NumberFormat from 'react-number-format'
+import { tailwind } from '@tailwind'
+import { ThemedTextV2, ThemedViewV2 } from './themed'
+import { View } from '.'
+import { ViewProps } from 'react-native'
+import { getNativeIcon } from '@components/icons/assets'
+import { translate } from '@translations'
+import { RandomAvatar } from '@screens/AppNavigator/screens/Portfolio/components/RandomAvatar'
+
+type SummaryTitleProps = React.PropsWithChildren<ViewProps> & ISummaryTitleProps
```

Comment:
We will pass children into this component?

Reply:
no children passed, removed `PropsWithChildren` and updated code